include/swift/AST/ClangModuleLoader.h

```diff
@@ -140,6 +140,7 @@ class ClangModuleLoader : public ModuleLoader {
   /// (The implementing `ClangImporter` class maintains separate Target info
   /// for use by IRGen/CodeGen clients)
   virtual clang::TargetInfo &getModuleAvailabilityTarget() const = 0;
+  virtual clang::TargetInfo &getTargetInfo() const = 0;
```

Comment:
Is this used?

Reply:
No, a vestige of a prior approach. Removed. Thank you.